tests/testcases.py

```diff
@@ -118,24 +124,28 @@ def stdout(self):
         clear_stdout was called."""
         return sys.stdout.getvalue()[self.stdout_start:].strip()
 
-    @patch('dusty.commands.utils.exec_docker')
+    def exec_docker_patch(self, *args):
+        args = ['/usr/local/bin/docker'] + [a for a in args]
```

Comment:
the subprocess can only user the `docker` alias if you set shell=True, but imo that is a security risk and adds some other complication as well